src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -187,7 +189,10 @@ internal MetricPointStatus MetricPointStatus
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public readonly long GetHistogramCount()
         {
-            if (this.aggType != AggregationType.Histogram && this.aggType != AggregationType.HistogramSumCount)
+            if (this.aggType != AggregationType.Histogram &&
```

Comment:
we can do a faster checking by checking if the bit for histogram is set.
for future PR :)

Reply:
nvm, not easy for aggregationtype.